src/bci_build/package/kubectl.py

```diff
@@ -0,0 +1,48 @@
+"""Kubectl Client BCI container"""
+
+from bci_build.container_attributes import SupportLevel
+from bci_build.os_version import CAN_BE_LATEST_OS_VERSION
+from bci_build.os_version import OsVersion
+from bci_build.package import ApplicationStackContainer
+from bci_build.package import ParseVersion
+from bci_build.package import Replacement
+from bci_build.package.versions import format_version
+from bci_build.package.versions import get_pkg_version
+
+KUBECTL_CONTAINERS = []
+
+KUBECTL_CONTAINERS = [
+    ApplicationStackContainer(
+        name="kubectl",
+        pretty_name="kubectl",
+        custom_description="Kuberneted CLI for communicating with a Kubernetes cluster's control plane, using the Kubernetes API.",
+        os_version=os_version,
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        version="%%kubectl_version%%",
+        tag_version=format_version(
+            kubectl_ver := get_pkg_version(f"kubernetes{ver}", os_version),
+            ParseVersion.MINOR,
+        ),
+        version_in_uid=False,
+        additional_versions=[format_version(kubectl_ver, ParseVersion.MAJOR)],
+        replacements_via_service=[
+            Replacement(
+                regex_in_build_description="%%kubectl_version%%",
+                package_name=f"kubernetes{ver}",
+                parse_version=ParseVersion.PATCH,
+            )
+        ],
+        package_list=[f"kubernetes-client{ver}"],
```

Comment:
the package is called `kubernetes{ver}-client`

Reply:
fixed.